src/TRE-UI/Views/Approval/EditMemberships.cshtml

```diff
@@ -113,15 +114,19 @@
                                         </div>
                                     }
                                 </td>
+                                    <td>
+                                        @Model[i].ProjectExpiryDate
+                                    </td>
                             </tr>
 
                         }
-                        <div class="hero-block-1">
-                            <button type="submit" class="btn btn-sm btn-primary mb-3"><i class="fa fa-save me-2"></i><span class="small">Save Changes</span></button>
-                        </div>
+                        
                     }
                     </tbody>
                 </table>
+                <div class="hero-block-1">
```

Comment:
Lines 127 to 129 need to be moved back with in the html form just above the } on line 124

Reply:
Done